Packs/GoogleThreatIntelligence/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,163 @@
+#### Integrations
+
+##### New: Google Threat Intelligence - ASM Issues
```

Comment:
The heading format for new content items should not include the "New:" prefix. Please remove it from this line and all similar headings throughout the file.